client/src/routes.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Switch, Route, Redirect } from 'react-router-dom';
+import CollectionPage from './pages/CollectionPage';
+import AuthPage from './pages/AuthPage';
+
+const UseRoutes = (isAuthenticated) => {
```

Comment:
This function is confusing.
`UseRoutes` returns JSX like a React component, but it's not possible to use it with `<UseRoutes />`.
The naming sounds like a custom hook, but it's in PascalCase.

You could do this:

```
const Routes = ({ isAuthenticated}) => {
```
with `<Routes isAuthenticated={false} />`

The filename doesn't fit too.